core/src/main/java/jenkins/model/Nodes.java

```diff
@@ -139,6 +140,7 @@ public void run() {
             });
             // TODO there is a theoretical race whereby the node instance is updated/removed after lock release
             persistNode(node);
+            NodeListener.fireOnCreated(node);
```

Comment:
Any particular concern for this listener or something nice to do for all listeners?